include/lbann/layers/io/input/generic_input_layer.hpp

```diff
@@ -36,6 +36,10 @@
 #include "lbann/callbacks/callback_imcomm.hpp"
 
 namespace lbann {
+
+/**
+ * @todo Move functionality to input_layer.
+ */
```

Comment:
Is this an actual todo? Pinging @bvanessen.

Reply:
I'm just being passive aggressive :p